index.html

```diff
@@ -241,5 +256,50 @@ <h2>Skills</h2>
         </ul>
       </div>
     </section>
+    <section class="form">
+      <p class="form-text">
+        <!-- I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. -->
+      </p>
+      <!-- This is the Form section of the Page with validations -->
+      <form class="form-inputs" action="https://formspree.io/f/xyyoqbgr" method="post">
+        <input type="text" class="form-name" name="name" placeholder="Full name" maxlength="30" required>
+        <input type="email" class="form-email" name="email" placeholder="Email address" required>
+        <textarea name="Message" class="form-comments" placeholder="Enter text here" cols="30" rows="10" maxlength="500" required></textarea>
+        <button type="submit" class="form-button">Get in touch</button>
+      </form>
+      <ul class="list2">
+        <li>
+          <a 
+            href="https://github.com/MarcoAurelioAntonio" 
+            class="github1"></a>
+        </li>
+        <li>
+          <a
+            href="https://www.linkedin.com/in/gustavo-porcilio-4496a223a/"
+            class="linkedin1"
+          ></a>
+        </li>
+        <li>
+          <a 
+            href="#" 
+            class="angelist1">
+          </a>
+        </li>
+        <li>
+          <a 
+            href="https://twitter.com/gustavoporcilio" 
+            class="twitter1"></a>
+        </li>
+        <li>
+          <a 
+            href="#" 
+            class="medium1">
+          </a>
+        </li>
+      </ul>
+    </section>
+    <div class="final">
+      <div class="final-line"></div>
+    </div>
   </body>
```

Comment:
- [ ] Kindly make sure your section design sticks to the one from Figma as required for the project.

| Figma design | Your Design |
| ------------------- | ------------------ |
| ![Screenshot from 2022-07-13 00-25-06](https://user-images.githubusercontent.com/83042742/178606752-7fff4bf6-6cf2-40c7-b806-ed9f2d0385cd.png) | ![Screenshot from 2022-07-13 00-24-03](https://user-images.githubusercontent.com/83042742/178606855-02f83402-1c4c-4f55-acfa-0fc96c0978d2.png) |

Reply:
@tresorsawasawa problem solved issues fixed.